### DISCUSSION TITLE
Update Multi-Site management

### DISCUSSION THREAD

**Discussion by yoannisj** [solspace/craft-calendar#124] [Feature Requests]
When a Calendar is enabled for multiple sites, the Events can be enabled/disabled on a per-site basis but it still requires that editors select a site in the site dropdown on the top left (and therefore reloading the page and leaving the Event editing form).

<img width="1564" alt="event-multi-site-fields" src="https://user-images.githubusercontent.com/3032696/164243451-e74c03af-d719-4976-9c68-d9a5107a2f8c.png">

This is similar to how built-in Element types for Craft-CMS were working before, but these can now be enabled/disabled for each site without leaving the element's editing form. Indeed, the "Enabled" field in the element sidebar on the right can be expanded to show the full list of sites which allows editors to create elements and enable them for specific sites before the first save.

<img width="1565" alt="entry-multi-site-fields" src="https://user-images.githubusercontent.com/3032696/164243503-711d458e-aa26-45cc-b638-1c29160cbb18.png">

It would be great if this Calendar Events were updated to use the updated site enabling/disabling mechanism; for consistency and also to simplify the workflow when creating new Events and fine-tuning their enabled/disabled state for each site.

**Comment by kjmartens** [comment]
Thanks for your feedback. We'll very likely be adjusting Calendar 4 for this. :)
